docs/api/xontribs_meta.rst

```diff
@@ -0,0 +1,11 @@
+.. _xonsh_xontribs_meta:
```

Comment:
There also needs to be an entry for this in `docs/api/index.rst`, I think

Reply:
Ok I will check

Reply:
about the failing amalgam, any clue ?

Reply:
Yep! See other inline comment

Reply:
I have updated the `index.rst` but I am not sure if that is the correct way.